projects/fsastorefrontlib/src/cms-components/banner/enriched-responsive-banner.component.spec.ts

```diff
@@ -34,6 +35,20 @@ describe('EnrichedResponsiveBannerComponent', () => {
       url: '/medias/logo-fsa.jpg',
     },
     urlLink: '/logo',
+    configStyles: JSON.stringify({
+      config: {
+        textBox: true,
+        textBoxType: 'box',
+        textBoxVerticalPosition: 'middle',
+        textBoxHorizontalPosition: 'left',
+        textBoxMaxWidth: '410px',
+        textBoxMargin: '0 0 0 17%',
+        textBoxPadding: '2vw',
+        textBoxTitle: true,
+        textBoxDetails: true,
```

Comment:
Maybe these properties are not needed at all? User could simply omit them from impex file and they shouldn't be displayed.